Common/GPU/Vulkan/thin3d_vulkan.cpp

```diff
@@ -848,8 +848,9 @@ VKContext::VKContext(VulkanContext *vulkan)
 		}
 
 		// Older ARM devices have very slow geometry shaders, not worth using.  At least before 15.
```

Comment:
At least before 18.

Reply:
Well, I don't know that at say 17 are slow, but they're apparently broken.

-[Unknown]